apps/cookbook/src/app/component-status/component-status.component.ts

```diff
@@ -432,6 +421,33 @@ export class ComponentStatusComponent implements OnInit, OnDestroy {
       })
     );
   }
+
+  getPageOfIssues(url: string) {
+    const options = {
+      headers: new HttpHeaders({
+        Authorization: 'token ' + environment.oauth.githubToken1 + environment.oauth.githubToken2,
+      }),
+      observe: 'response' as const,
+    };
+    return this.http.get<Observable<any>>(url, options).pipe(
```

Comment:
The generic type should be an array as it was before (see `getGithubEnhancementIssues` before change: `http.get<any[]>`).
👉🏻  This ensures the mapping operator has an array type in `res.body`.
 It can be further typed by using an explicit type for the Github issues or at least `Object[]`:
```suggestion
    return this.http.get<Object[]>(url, options).pipe(
```